torch/profiler/_utils.py

```diff
@@ -1,7 +1,9 @@
 from collections import deque
 from dataclasses import dataclass
 from typing import Dict, List
+import re
 
+import torch
```

Comment:
Don't `import torch` inside library files since it can lead to a circular import.

Reply:
But without importing it, how can I use torch.mean, torch.tensor? Do I have to import all of them in?
Checking with other file in torch/profiler, they all did import torch, so that's why I follow the pattern :(

Reply:
I take it back. While it isn't ideal, basically every other file also does it so I guess that battle has been lost. If that weren't the case, the way you get around it is by importing outside global scope (e.g. in a function body).

Reply:
👍